v0/pipeline/steps/cd/k8s-route-spec.yaml

```diff
@@ -0,0 +1,30 @@
+title: RouteSpec
+type: object
+required:
+  - route
+properties:
+  route:
+    type: object
+    required:
+      - type
+      - rules
+    properties:
+      type:
+        type: string
+        enum:
+          - http
+          - tcp
```

Comment:
are we planning to support all these types in first phase? @markoharness

Reply:
@alebuh please remove all route type enums except "http". For now lets go only with HTTP.